atomic_reactor/utils/resource_manager.py

```diff
@@ -6,193 +6,309 @@
 of the BSD license. See the LICENSE file for details.
 """
 
-import os
-import logging
-import json
-import paramiko
 import backoff
+import json
 import jsonschema
-from typing import List, Any, Dict, Tuple
+import logging
+import paramiko
+import random
+import os
+from contextlib import contextmanager
 from datetime import datetime
 from shlex import quote
-
-PARAMIKO_MAX_RETRIES = 3
-PARAMIKO_BACKOFF_FACTOR = 1
-
-ROOT_PATH = 'resource_manager/'
+from typing import List
 
 RESOURCE_LOCK_SCHEMA = 'schemas/resource-lock-content.json'
+SSH_COMMAND_TIMEOUT = 45
+TIMEOUT_FAIL_BUILD = 3000
+RELATIVE_PATH = 'resource_manager'
```

Comment:
@lkolacek What is the relative path of this RELATIVE_PATH? Is the working directory changed at somewhere or is the working directory determined by the configured ssh login?